app/src/main/java/ar/com/wolox/android/example/ui/session/SessionActivity.java

```diff
@@ -0,0 +1,32 @@
+package ar.com.wolox.android.example.ui.session;
+
+import android.widget.Toast;
+
+import javax.inject.Inject;
+
+import ar.com.wolox.android.R;
+import ar.com.wolox.android.example.ui.login.LoginActivity;
+import ar.com.wolox.android.example.utils.UserSession;
+import ar.com.wolox.wolmo.core.activity.WolmoActivity;
+
+/**
+ * Session Activity
+ */
+public class SessionActivity extends WolmoActivity {
+
+    @Inject
+    UserSession userSession;
+
+    @Override
+    public int layout() {
+        return R.layout.activity_base;
+    }
+
+    @Override
+    public void init() {
+        if (userSession.getUserIsLogged()) {
```

Comment:
you are always sending the user to loginActivity. You should create your homeActivity (with fragment, view, presenter and module) And in the fragment you can put a text that say: HomeFragment, in the middle of the view. So you can redirect there if you are already logged. Your card of navegation also want you to close the app on back pressed if you are already logged, so you must implement all the views.

Reply:
Same here